README.md

```diff
@@ -19,3 +19,10 @@ python setup install
 --------
 Use freely all the functions and/or two easy commands (lc-get and lc-submit).
 * Please input your own username and password in func.py if you want to use submit functions.
+* Please change the language in func.py's get_problem_source method for another language. For example, if you want to download java problems change to this: "def get_problem_source(self, problem, language='Java'):"
+
+Explanation for lc-refresh command:(added on 05/05/2016 by henryhoo)
```

Comment:
Please remove the date and name in the README. It will be kept in the git history.